File: jersey/src/main/java/org/jersey/service/EchoService.java
package org.jersey.service;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.PathParam;

import lombok.extern.slf4j.Slf4j;

@Slf4j
@Path("/echo")
public class EchoService {
	@GET
	@Path("/{message}")
	public String echo(@PathParam("message") String message) {
		log.debug(message);
		return message;
	}
}

File: jersey/src/main/javascript/app/services/echoService.js
app.service('echoService', function($http, $rootScope, $window, $log) {
this.echo = function(message, callback) {
    $http({
        method: 'POST',
        url: '/jwt-authentication/api/rest/echo',
        data: message
     }).success(function(data){
    	 callback(data.message);
    }).error(function (data, status, headers, config) {
    	$log.debug("error with echoService");
    });
 }
});
